src/smartStrategy.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+import { UnauthorizedError } from 'fhir-works-on-aws-interface';
+
+export interface AuthStrategy {
+    /**
+     * Validates the JWT access_token
+     * @param accessToken
+     * @throws UnauthorizedError
+     */
+    isTokenValid(accessToken: string): Promise<void>;
+}
+
```

Comment:
Add a comment explaining this strategy; something like:
```suggestion
/**
  * This strategy uses the /userinfo endpoint to verify the `access_token` and get the `fhirUser` claim for further authorization. userinfo endpoint is OIDC compliant: https://openid.net/specs/openid-connect-core-1_0.html#UserInfo
  */
```

Reply:
Thanks for the suggestion, added!